assembl/tests/views_tests/test_graphql.py

```diff
@@ -738,11 +797,13 @@ def test_mutation_create_post_without_subject(graphql_request, thematic_and_ques
             }}}
 
 
+@freeze_time("2018-3-1")
 def test_mutation_create_post_on_column(graphql_request,
                                         test_session,
-                                        idea_message_column_positive):
-    idea_id = to_global_id('Idea', idea_message_column_positive.idea_id)
-    classifier = idea_message_column_positive.message_classifier
+                                        idea_message_column_positive_on_thread_phase):
+    idea_id = to_global_id('Idea', idea_message_column_positive_on_thread_phase.idea_id)
```

Comment:
how is it possible to have a positive column on a thread positive ? a column is only possible in multicolumn phase

Reply:
There's no constraint in the backend